src/main/java/store/bookstoreapp/model/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package store.bookstoreapp.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cards_items")
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "shopping_cart_id")
+    private ShoppingCart shoppingCart;
+    @ManyToOne
+    @NotNull
+    @JoinColumn(name = "book_id")
+    private Book book;
+    @NotNull
+    private Integer quantity;
+    @Column(nullable = false)
```

Comment:
@NotNull is annotation for validation

Asked constraints to database, e.g. @Column or @JoinColumn